<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mein einfacher Listeneditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item row="2" column="4">
     <widget class="QPushButton" name="delButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Löschen</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QListWidget" name="itemList">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="delAllButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Alle löschen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="countLabel">
      <property name="text">
       <string>Anzahl:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="addButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Hinzufügen</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLCDNumber" name="countLcdNumber">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalButtonSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="editButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Bearbeiten</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>itemList</zorder>
   <zorder>addButton</zorder>
   <zorder>delButton</zorder>
   <zorder>countLabel</zorder>
   <zorder>countLcdNumber</zorder>
   <zorder>delAllButton</zorder>
   <zorder>horizontalButtonSpacer</zorder>
   <zorder>lineEdit</zorder>
   <zorder>editButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Laden...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Speichern...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Über...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>itemList</tabstop>
  <tabstop>delButton</tabstop>
  <tabstop>delAllButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>addButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>342</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
